tests/wamr-test-suites/spec-test-script/all.py

```diff
@@ -105,7 +103,10 @@ def test_case(
     xip_flag=False,
     clean_up_flag=True,
     verbose_flag=True,
+    qemu_flag=False,
+    qemu_firmware='',
 ):
+    global IWASM_CMD
```

Comment:
Don't suggest to change `IWASM_CMD`.

Maybe adding a new `QEUMU_CMD` instead ? and change L128 like 
`CMD.append(QEMU_CMD if qemu_flag else (IWASM_CMD if not sgx_flag else IWASM_SGX_CMD))`

Reply:
OK